src/main/java/org/folio/inventory/dataimport/exceptions/DuplicatedEventException.java

```diff
@@ -0,0 +1,8 @@
+package org.folio.inventory.dataimport.exceptions;
```

Comment:
We can use DuplicateEventException from folio-kafka-wrapper instead of creating new one

Reply:
Good point! Thank you. Done.